cfme/configure/configuration/region_settings.py

```diff
@@ -18,6 +18,112 @@
 from cfme.utils.update import Updateable
 
 
+# =======================================TAGS=============================================
+
+table_button_classes = [Button.DEFAULT, Button.SMALL, Button.BLOCK]
+
+
+class CompanyTagsAllView(RegionView):
+    """Company Tags list view"""
+    category_dropdown = BootstrapSelect('classification_name')
+    table = DynamicTable(locator='//div[@id="classification_entries_div"]/table',
+                         column_widgets={
+                             'Name': Input(id='entry_name'),
+                             'Description': Input(id='entry_description'),
+                             'Actions': Button(title='Add this entry',
+                                               classes=table_button_classes)},
+                         assoc_column='Name', rows_ignore_top=1, action_row=0)
+
+    @property
+    def is_displayed(self):
+        return (
+            self.company_tags.is_active() and
+            self.table.is_displayed
+        )
+
+
+@attr.s
+class Tag(Pretty, BaseEntity, Updateable):
+    """ Class represents a category in CFME UI
+        Args:
```

Comment:
**REQUIRED**
There should be a blank line after the summary.

Reply:
corrected